utils/io/fileutils/files.go

```diff
@@ -133,6 +133,32 @@ func ListFilesRecursiveWalkIntoDirSymlink(path string, walkIntoDirSymlink bool)
 	return
 }
 
+// Return the list of files in the specified path that has the extension
+func ListFilesWithSpecificExtension(path, ext string) ([]string, error) {
+	sep := GetFileSeparator()
+	if !strings.HasSuffix(path, sep) {
+		path += sep
+	}
+	fileList := []string{}
+	files, _ := ioutil.ReadDir(path)
+	path = strings.TrimPrefix(path, "."+sep)
+
+	for _, f := range files {
+		if !strings.HasSuffix(f.Name(), ext) {
+			continue
+		}
+		filePath := path + f.Name()
+		exists, err := IsFileExists(filePath, false)
+		if err != nil {
+			return nil, err
+		}
+		if exists || IsPathSymlink(filePath) {
```

Comment:
I belive that you are interested in finding only files or symlinks to files.
I'm not 100% sure, but there's a chance that the above line will also find symlinks to directories. Worth checking just to be sure.

Reply:
I've changed it to ignore directories.